### DISCUSSION TITLE
Use `gap` CSS property with Flexbox based components such as <Group />

### DISCUSSION THREAD

**Discussion by dallanlee** [mantinedev/mantine#373] [Feedback]
What is the intended browser support for Mantine? In the hooks section it says browser support is all major modern browsers. If this is the case for components as well, I would suggest refactoring the styling of Flexbox-based items to use [`gap`](https://caniuse.com/flexbox-gap) instead of the current `margin` hack. https://caniuse.com/flexbox-gap

**Comment by rtivital** [comment]
Sounds good, we do not support IE11 and all other browsers seem to have a fine support. Can I interest you in sending a PR with this feature? 😉

**Reply by dallanlee** [reply]
Okay, good to know. Thanks. I'm not sure I'm familiar enough yet with all the layout components to know the nuances of how they're using Flexbox to be able jump into it. I'm slowly working through all of Mantine to determine if we—at my place of work—will use it as our base library. I'd like to work on a PR but I'm just not sure about the timeframe. Let me get back to you tomorrow or sometime this week.

**Reply by rtivital** [reply]
Alright, no worries, I'll do it myself then. Thanks for proposal – it will solve some of our issues with group

**Reply by TKDev7** [reply]
@dallanlee Might want to open an issue as well.

**Reply by rtivital** [reply]
Already added this as part of 3.2 release, no need to create an issue
